### ISSUE TITLE
Form validation with many elements takes a long time

### ISSUE THREAD

**Issue by kabacinsp** [rjaros/kvision#313]
I noticed that the ```validate``` method from the ```Form``` class causes very large delays. After conducting the research, I noticed that the reason is ```refresh``` the component while assigning a value to the ```validatorError``` field. This process takes from 180 ms to 340 ms, which, with a large number of components, gives a feeling of freezing the application. 
Is there any way to optimize this process?

**Comment by rjaros** [comment]
Please try to use `singleRender` wrapper and let me know, if it makes a difference:
```
                    formPanel.singleRender {
                        formPanel.validate()
                    }
```

**Comment by kabacinsp** [comment]
> Please try to use `singleRender` wrapper and let me know, if it makes a difference:
> 
> ```
>                     formPanel.singleRender {
>                         formPanel.validate()
>                     }
> ```

It solved my problem

**Comment by rjaros** [comment]
I'll add this internally in the next release. Please leave the issue open for now.

**Comment by rjaros** [comment]
Fixed in 5.3.0
